pkg/base/capacityplanner/node_capacity.go

```diff
@@ -32,6 +32,11 @@ type reservedCapacity struct {
 	StorageClass string
 }
 
+// String is pretty print function for reservedCapacity
+func (rc *reservedCapacity) String() string {
+	return fmt.Sprintf("{size: %d, SC: %s}", rc.Size, rc.StorageClass)
```

Comment:
what about to use bytesize formatter for logging? This for example github.com/inhies/go-bytesize for nicely formatted string representation of the size.

Reply:
Lets keep consistent.